### PR TITLE
[DOCKER/VISION] 

### PR DESCRIPTION
**Pull request by Alestrio**
Edits :
- CI/CD building docker image for x86_64 and arm64 arch on push on main
- Fixed vision that I broke
- Support for custom prompts (in database folder)

### CODE REVIEW

**Comment by Paillat-dev** [comment]
I'm gonna check that later when I have time

File: code/vision_processing.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 try:
     client = vision.ImageAnnotatorClient()
 except:
-    debug("Google Vision API is not setup, please run /setup")
+    print("Google Vision API is not setup, please run /setup")
```
Comment (Paillat-dev) [inline code/vision_processing.py:13]: Google vision API is global, should be `add your google credentials file in the /data/google-vision/botator.json file.`

File: .github/workflows/buildDockerImage.yml
Comment (Paillat-dev) [inline .github/workflows/buildDockerImage.yml:1]: Trusting you for that one

File: code/config.py [hunk 2]
```diff
@@ -20,6 +20,8 @@ def debug(message):
     # if the os is windows, we logging.info(message), if
     if os.name == "nt":
         logging.info(message)
+    else:
```
Comment (Paillat-dev) [inline code/config.py:23]: Why? It's supposed to debug only when testing. I should add a .env debug variable instead.

**Review by Paillat-dev — CHANGES_REQUESTED** [review]

File: code/config.py (hunk 2 above)
Comment (Alestrio) [reply code/config.py:23]: Yup, definitely
And as for the reason, it's to print errors on stdin while in docker container

File: code/vision_processing.py (hunk 1 above)
Comment (Alestrio) [reply code/vision_processing.py:13]: Was just a way to make it not mandatory, still works the same way, but doesn't crash if the cred file isn't there

**Pull request merged**